creator/refferalTokens/models.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+from django.db import models
+from creator.studies.models import Study
+
+User = get_user_model()
+
+
+class ReferralToken(models.Model):
+    """
+    An Event that involved some user, study, file, version, or a combination
+    of them.
+    """
+
+    class Meta:
+        permissions = [
+            ("add_refferaltoken", "Can generate a new referral token",)
+        ]
+
+    id = models.UUIDField(default=uuid.uuid4, unique=True, editable=False)
+
+    email = models.CharField(
```

Comment:
We should use the [EmailField](https://docs.djangoproject.com/en/3.0/ref/models/fields/#emailfield) here.